decidim-bulletin_board-app/app/assets/javascripts/sandbox/vote.es6

```diff
@@ -50,13 +55,41 @@ $(async () => {
         invalidVoteFn();
       }
     },
-    onVoteEncrypted({ encryptedVote }) {
+    castOrAuditBallot() {
+      $encryptVote.prop("disabled", true);
+      $castVote.show();
+      $auditVote.show();
+    },
+    onAuditVote(onEventTriggered) {
+      $auditVote.on("click", onEventTriggered);
+    },
+    onVoteValidation(onEventTriggered) {
+      $castVote.on("click", onEventTriggered);
+    },
+    onVoteAudition(auditedVote, auditFileName) {
+      vote = JSON.stringify(auditedVote);
+
+      let link = document.createElement("a");
+      link.setAttribute("href", `data:text/plain;charset=utf-8,${vote}`);
+      link.setAttribute("download", auditFileName);
+      document.body.appendChild(link);
+      link.click();
+      document.body.removeChild(link);
+    },
+    onAuditComplete() {
+      $castVote.prop("disabled", true);
+      $auditVote.prop("disabled", true);
+
+      $auditMessage.show();
+      $vote.css("background", "green");
+    },
+    onVoteEncrypted({ encryptedBallot }) {
       return $.ajax({
         method: "POST",
         contentType: "application/json",
         data: JSON.stringify({
           voter_id: $voterId.val(),
-          encrypted_vote: encryptedVote,
+          encrypted_vote: encryptedBallot,
```

Comment:
I think that we should change the `encrypted_vote` to `encrypted_ballot` to make it consistent.